sample_apps/simple_app/example_app.mak

```diff
@@ -28,12 +28,13 @@ S= $(CERTIFIER_ROOT)/src
 O= $(OBJ_DIR)
 US=.
 I= $(CERTIFIER_ROOT)/include
-INCLUDE= -I$(I) -I/usr/local/opt/openssl@1.1/include/ -I$(S)/sev-snp/
+INCLUDE= -I$(I) -I/usr/local/opt/openssl@1.1/include/ -I$(S)/sev-snp/ -I.
+COMMON_SRC = ../common
```

Comment:
Minor: Could you move to specifying dirs off of `CERTIFIER_ROOT`; i.e. this becomes

`COMMON_SRC = $(CERTIFIER_ROOT)/sample_apps/common`

May make it easier to follow dir-paths.